frontend/src/configs/local.js

```diff
@@ -1,11 +1,16 @@
-const { PLAUSIBLE_DATA_DOMAIN_STAGING } = require("./common");
-
 // (thuang): For local development, please copy the content of this file
 // to a new file named `configs.js` in this directory.
 
 const configs = {
+  // Dev
```

Comment:
@seve thanks for pointing this out!

I just added different endpoints in `local.js`, so we can still copy this to `configs.js` locally and run `npm run dev` without using `make local-sync`. So this should address our `npm run dev` use case now?

Thanks!

Reply:
Very much a nit, but the process of:
```
cp src/configs/dev.js src/configs/configs.js
npm run dev
```
is **way** faster than
```
cp src/configs/local.js src/configs/configs.js
code src/configs/configs.js
# uncomment one line and comment another line
npm run dev
```

Reply:
yeah I actually just changed the default in `local.js` to be pointing to the dev API

Reply:
oh missed that part, sounds good!